### PR TITLE
ignorePlurals: accept a slice of string for languages

### PR DESCRIPTION
**Pull request by ercolanelli-leo**

### CODE REVIEW

File: algoliasearch/types_settings.go [hunk 1]
```diff
@@ -38,7 +38,7 @@ type Settings struct {
 	HighlightPostTag           string      `json:"highlightPostTag"`
 	HighlightPreTag            string      `json:"highlightPreTag"`
 	HitsPerPage                int         `json:"hitsPerPage"`
-	IgnorePlurals              bool        `json:"ignorePlurals"`
+	IgnorePlurals              interface{} `json:"ignorePlurals"`
```
Comment (aseure) [inline algoliasearch/types_settings.go]: Could you add a comment here (like for `distinct` and `removeStopWords`)?

Comment (ercolanelli-leo) [inline algoliasearch/types_settings.go]: done

Comment (aseure) [inline algoliasearch/types_settings.go]: LGTM now 👍

**Comment by aseure** [comment]
@ercolanelli-leo I've added a commit to fix the test. It was because `ignorePlurals` wasn't handled properly in `Settings.ToMap`. It's done now. If you think it's OK, feel free to merge it. ;)

**Comment by ercolanelli-leo** [comment]
LGTM :+1:

**Pull request merged**